pkg/config/configuration.go

```diff
@@ -123,3 +138,26 @@ func LoadKubernetesConfiguration(config, namespace string, operatorClient operat
 	}
 	return &conf, nil
 }
+
+func IsSecretAllowed(storeName, key, defaultSecretAccess string, allowedSecrets, deniedSecrets map[string]map[string]struct{}) bool {
```

Comment:
missing data type for allowedSecrets?

Reply:
nope same as deniedSecrets. Changed logic.